### ISSUE TITLE
Similar Model - During Training, Predictions are not close to targets

### ISSUE THREAD

**Issue by ghost** [xumingxingsjtu/STTN#2]
I am trying to implement a similar model but changing GCN to a GAT in your model - however, during training, I don't seem to capture the trends in the input data - all the predictions from the final layer are close to the mean value, and do not follow any trend from the input - MAE is hence very high compared to the published MAEs in your paper. 

Could you please help with a one-liner comment on the two questions below?

1 - When training, how do the predictions from the final layer of the model look? Do they capture the trend of the input data or produce values close to the mean value of the training data?
2 - During training, what does the training loss look like in your experiments - is it lower than the final MAE reported for 60 min interval? 

during training, the final layer outputs (n_timesteps, n_sensors, 1) value in one go while in testing, it does 1, n_sensors,1 for the horizon number of timesteps. In general, test loss and training loss need to converge with epochs of training, but test loss is generally slightly higher than the training loss - does that apply to this auto-regressive model?
